### DISCUSSION TITLE
UNPKG and releases

### DISCUSSION THREAD

**Discussion by EttinCon** [Spiderpig86/Cirrus#64] [Q&A]
Hi! We've been checking a few things to ensure we're not using outdated classes or files, and noticed that the UNPKG url redirects to v0.6.2 and the file it returns is labelled v0.6.1. 

![image](https://user-images.githubusercontent.com/32328287/123738662-917f8d80-d8e8-11eb-91e5-3160a031d8d3.png)

Is this working as expected, and if so, what's the best way for us to stay apace with changes so that we don't break anything, please?

**Comment by Spiderpig86** [comment]
Hi, I just checked the commit history regarding all the changes and the file here indeed has all the latest changes. The only issue is the 0.6.1 error here. I will get this patched with the correct versioning. Thanks for pointing it out!

**Reply by EttinCon** [reply]
Thanks! Are there documentation updates in progress, or should we let you know if we find any old references?

**Reply by Spiderpig86** [reply]
The docs should be up to date with all the latest additions. Should there be an error, please let me know!

**Reply by EttinCon** [reply]
Will do, thanks!
